src/core/analysis/mcmc/Mcmcmc.cpp

```diff
@@ -1136,14 +1136,14 @@ void Mcmcmc::setHeatsInitial(const std::vector<double>& ht)
         throw RbException()<<"Heat vector contains "<<ht.size()<<" heats, but there are "<<num_chains<<" chains.";
 
     // 2. Check that the heats are non-zero
-    for(int i=0; i<int(ht.size()); i++)
+    for(size_t i=0; i<ht.size(); i++)
```

Comment:
I think this is wrong.  The point here is to convert `ht.size()` to a signed type, so that if `ht.size() == 0` the result doesn't overflow.

Reply:
Ah, okay. I'm not sure if you can undo the merge, but if so, I'm happy to push a commit to restore `int` here (and the other place you point out below). If not, one of us can open another PR to that effect.

Reply:
Actually this one is fine.  Its the one below (`ht.size()-1`) that is the issue.